src/api/api_routes.js

```diff
@@ -32,6 +32,9 @@ module.exports = [
     ['GET', '/v1/api/wallet/hash', 'WalletController@hash'],
     ['GET', '/v1/api/wallet/address', 'WalletController@address'],
     ['GET', '/v1/api/wallet/getWalletInfo', 'WalletController@getWalletInfo'],
+    ['GET', '/v1/api/wallet/getTokenBalances', 'WalletController@getTokenBalances'],
```

Comment:
We don't need this endpoints in the API server, they should be handled by the Proxy api.